kuma/static/js/components/user-signup/auth-modal.js

```diff
@@ -3,38 +3,44 @@
 
     var authModalContainer = document.getElementById('auth-modal');
 
-    if (!window.mdn.modalDialog.showModal(authModalContainer)) {
+    if (
+        !authModalContainer ||
+        !window.mdn.multiAuthEnabled ||
+        !window.mdn.modalDialog.shouldShowModal()
+    ) {
         return;
     }
 
-    /* The React header takes just a couple of milliseconds to
-       long to load so, we need to wrap the code below in a 
-       setTimeout */
-    setTimeout(function() {
```

Comment:
Much of this file's diff gets very complicated. It's related to my deleting the line  `setTimeout(function() {` and `}, 100);` which don't make any sense to have any more now that this .js is loaded with `defer`.